driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverManagerConfig.java

```diff
@@ -96,7 +96,7 @@ private void initBrowser() {
         }
 
         if (StringUtils.isBlank(this.getPlatformName().orElse(null))) {
-            this.setPlatformName(PropertyManager.getProperty(TesterraProperties.BROWSER_PLATFORM, null));
+            this.setPlatformName(PropertyManager.getProperty(TesterraProperties.BROWSER_PLATFORM, ""));
```

Comment:
The platform name getter is an `Optional.ofNullable()` but `""` is not null and therefore `Optional.isPresent()` is `true` with an empty string.

Reply:
Please put that check into the setter.